help/de-de.markdown

```diff
@@ -233,14 +233,21 @@ des Screenshots, den Sie behalten wollen. Sie können die Größe des ausgewähl
 Bereichs ändern, wie bei jedem anderen Element.<br>
 Wenn Sie mit Ihrer Auswahl zufrieden sind, klicken Sie die Bestätigen-Schaltfläche in
 der Symbolleiste oder drücken Sie die <kbd>Enter</kbd>-Taste. Sie können den Vorgang
-abbrechen, indem Sie die Abbrechen-Schaltfläche klicken oder <kbd>ESC</kbd> drücken.
+abbrechen, indem Sie die Abbrechen-Schaltfläche klicken oder <kbd>ESC</kbd> drücken.<br>
+<kbd>Strg</kbd> + <kbd>-</kbd> schneidet den Bereich sofort auf die vorhandenen Objekte zu.
 </p>
 <p class="hint">
 <em>Auto-Crop</em>: Wenn Sie einen einfarbigen Rahmen von Ihrem Screenshot entfernen möchten,
 wählen Sie einfach <em>Automatisch zuschneiden</em> aus dem Menü <em>Bearbeiten</em> und
 Greenshot wählt automatisch einen Bereich für den Zuschnitt aus.
 </p>
 
+<a name="editor-enlarge"></a>
+<h3>Screenshot vergrössern</h3>
+<p>
+Wenn Sie den Screenshot etwas vergrössern wollen, drücken Sie <kbd>Shift</kbd> + <kbd>+</kbd>, was auf allen vier Seiten je 25 Pixel hinzufügt.
```

Comment:
```suggestion
Wenn Sie den Screenshot etwas vergrößern wollen, drücken Sie <kbd>Shift</kbd> + <kbd>+</kbd>, was auf allen vier Seiten je 25 Pixel hinzufügt.
```

Reply:
perfect. there is no ß in Switzerland, therefore they are all missing.... I thought to look at them, but my eyes are not trained for...